### PR TITLE
22970: Update OMBInfo to be specific to 1990s

### PR DESCRIPTION
**Pull request by elder-joseph**
## Description
As a developer, I need to update the VRRAP OMB content that so that it is compliant with the registration of the form with OMB.

[Originating Issue](https://github.com/department-of-veterans-affairs/va.gov-team/issues/22970)

## Testing done
Testing passes locally.

## Screenshots
Screen Shot 2021-04-09 at 11 48 52 AM
Screen Shot 2021-04-09 at 2 08 31 PM

## Acceptance criteria
- [x] The respondent burden on the introduction page is: 10 minutes
- [x] The OMB Control # on the introduction page is: 2900-0885
- [x] The Expiration Date on the introduction page is: October 31, 2021
- [x] The content in the privacy act statement modal is:
```
Privacy Act Statement {h3}

Privacy Act Notice: This form under §8006 of Public Law 117-2 (H.R. 1319) authorizes VA to implement the Veteran Rapid Retraining Assistance Program (VRRAP) which provides eligible Veterans impacted by the COVID-19 pandemic with the opportunity to pursue a covered program of education in a high technology - high demand occupation. The responses you submit are considered confidential (38 U.S.C. 5701). VA may disclose the information that you provide, including Social Security numbers (SSN), outside VA if the disclosure is authorized under the Privacy Act, including the routine uses identified in the VA system of records, 58VA21/22/28, Compensation, Pension, Education, and Vocational Rehabilitation and Employment Records - VA, published in the Federal Register. The requested information is considered relevant and necessary to determine maximum benefits under the law. Your obligation to respond is required in order to obtain or retain education benefits. Giving us your SSN account information is voluntary. Refusal to provide your SSN by itself will not result in the denial of benefits. The VA will not deny an individual benefits for refusing to provide his or her SSN unless the disclosure of the SSN is required by Federal Statute of law enacted before January 1, 1975 and still in effect. Any information provided by applicants may be subject to verification through computer matching programs with other agencies.

Respondent Burden: We need this information to determine your ability to participate in the Veteran Rapid Retraining Assistance Program, 38 U.S.C. 3702 (d) and 38 CFR 36.4344. Title 38, United States Code, allows us to ask for this information. We estimate that you will need an average of 10 minutes to review the instructions, find the information, and complete this form. VA cannot conduct or sponsor a collection of information unless a valid OMB control number is displayed. You are not required to respond to a collection of information if this number is not displayed. Valid OMB control numbers can be located on the OMB Internet Page at www.reginfo.gov/public/do/PRAMain. If desired, you can call 800-827-1000 to get information on where to send comments or suggestions about this form.
```

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x]  No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/edu-benefits/1990s/content/OMBInfo.jsx [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+          matching programs with other agencies.
+        </p>
+      )}
+
+      <p>
+        <strong>Respondent Burden:</strong> We need this information to
+        determine your ability to participate in the Veteran Rapid Retraining
+        Assistance Program, 38 U.S.C. 3702 (d) and 38 CFR 36.4344. Title 38,
+        United States Code, allows us to ask for this information. We estimate
+        that you will need an average of 10 minutes to review the instructions,
```
Comment (shawkey-daniel) [inline src/applications/edu-benefits/1990s/content/OMBInfo.jsx:56]: If we are checking that minutes exists, shouldn't we use it to display how many?

**Review by shawkey-daniel — CHANGES_REQUESTED** [review]

File: src/applications/edu-benefits/1990s/content/OMBInfo.jsx (hunk 1 above)
Comment (elder-joseph) [reply src/applications/edu-benefits/1990s/content/OMBInfo.jsx:56]: That seems like a good approach, I can update it to include the minutes value there.

**Review by shawkey-daniel — APPROVED** [review]
LGTM

**Comment by elder-joseph** [comment]
Associated PR: https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/va.gov-team/22970

**Review by delli-gatti-michael — APPROVED** [review]

**Pull request merged**